src/lib/verifier/prod.ml

```diff
@@ -308,18 +256,15 @@ module Worker = struct
         ; verify_commands =
             f
               ( [%bin_type_class:
-                  User_command.Verifiable.t With_status.Stable.Latest.t
```

Comment:
This thing is being serialized, and unversioned. Wouldn't this be a breaking change?

Reply:
This is unversioned precisely because prover is always using the same binary as the main thread. It's a process that's spawned on start of the application. So it's not possible to have versions diverging in this context.

Reply:
I suggest we document this near the PRC :)